integration_tests/light_client_proxy/auth.sh

```diff
@@ -0,0 +1,152 @@
+set -euo pipefail
+source integration_tests/cli/common.sh
+
+random_string user
+cmd="(echo $passphrase; echo $passphrase) | dcld keys add $user"
+result="$(bash -c "$cmd")"
+user_address=$(echo $passphrase | dcld keys show $user -a)
+user_pubkey=$(echo $passphrase | dcld keys show $user -p)
+
+random_string user2
+cmd="(echo $passphrase; echo $passphrase) | dcld keys add $user2"
+result="$(bash -c "$cmd")"
+user_address2=$(echo $passphrase | dcld keys show $user2 -a)
+
+# 1. check non-existent values when no entry added via light client
```

Comment:
Should the comment be:
"check non-existent values via light client when no entry added"?

Reply:
Done